caffe2/operators/top_k_radix_selection.cuh

```diff
@@ -104,6 +104,7 @@ struct TopKTypeConfig<long> {
   typedef unsigned long long int RadixType;
 
   static inline __device__ RadixType convert(long v) {
+    //static_assert fails on windows, so leave it as CUDA_KERNEL_ASSERT
```

Comment:
...how does the CUDA_KERNEL_ASSERT not fail on windows?

Reply:
It will fail, if someone tries to run caffe2 radix sort for long inputs on windows pytorch build. Hopefully, no one will do it, but I can't make it static_assert, because with static_assert windows build itself fails.

Reply:
Should we just make this into `int64_t`?

Reply:
I'm not familiar with caffe2 code and don't know if it's possible. It is also probably untested?

Reply:
I don't think it will break anything because the assert `sizeof(long) == 8` here.